compiler/control/OMROptions.cpp

```diff
@@ -691,6 +691,7 @@ TR::OptionTable OMR::Options::_jitOptions[] = {
    {"enableIVTT",                         "O\tenable IV Type Transformation", TR::Options::enableOptimization, IVTypeTransformation, 0, "P"},
    {"enableJCLInline",                    "O\tenable JCL Integer and Long methods inlining", SET_OPTION_BIT(TR_EnableJCLInline), "F"},
    {"enableJITaaSHeuristics",             "O\tenable JITaaS heuristics", SET_OPTION_BIT(TR_EnableJITaaSHeuristics), "F"},
+   {"enableJITaaSDoubleCompile",          "O\tenable JITaaS to do both local and remote compilations", SET_OPTION_BIT(TR_EnableJITaaSDoubleCompile), "F"},
```

Comment:
Options have to be order alphabetically, so I think `enableJITaaSDoubleCompile` needs to come before `enableJITaaSHeuristics`.